organisaatio-ui/src/components/Sivut/Ryhmat/Muokkaus/MuokkausLomake.tsx

```diff
@@ -32,14 +26,26 @@ export type MuokkausLomakeProps = {
     handleTallenna: () => void;
 };
 
+export type InputBindProps = {
+    bind: TranslatedInputBind;
```

Comment:
Itse olisin ehkä antanut ton pelkän bind:in propsina ja hoitanut otsikon lokalisoinnin itse komponentissa. -> ei olisi tarvinnut uutta tyyppiä Propseja varten ja komponentti olisi "omavaraisempi".